### DISCUSSION TITLE
install deepin 23 v

### DISCUSSION THREAD

**Discussion by AhmedKhaledp-0** [linuxdeepin/developer-center#4715] [Troubleshooting | 故障排查]
### SUMMARY | 问题概要

I have a problem installing deepin 23 beta  
[deepin-installer.log](https://github.com/linuxdeepin/developer-center/files/11781754/deepin-installer.log)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

```plain
deepin 23 beta
```

### STEPS TO REPRODUCE | 复现步骤

1.i downloaded the iso file 
2.i tried to install many times 
3i thought the problem was with iso so i downloaded it again 
4 i got the same error

### OBSERVED RESULT | 观察到的结果

Did not install 

### EXPECTED RESULT | 期望的结果

Installed successfully 

### ADDITIONAL INFORMATION | 额外补充

[deepin-installer.log](https://github.com/linuxdeepin/developer-center/files/11781753/deepin-installer.log)
Its my install log

**Comment by BLumia** [comment]
```
2023-05-21, 01:41:28.079 [Info   ] [Process.cpp          Process::onReadInfo                 24] '/boot/efi/EFI/deepin/grub.cfg' -> '/boot/efi/EFI/boot/grub.cfg'
'/boot/efi/EFI/deepin/grub.efi' -> '/boot/efi/EFI/boot/grub.efi'
'/boot/efi/EFI/deepin/grubx64.efi' -> '/boot/efi/EFI/boot/grubx64.efi'
2023-05-21, 01:41:28.081 [Warning] [Process.cpp          Process::onReadWarning              31] cp: cannot create regular file '/boot/efi/EFI/boot/bootia32.efi-1684633288.bak'
2023-05-21, 01:41:28.081 [Warning] [Process.cpp          Process::onReadWarning              31] : No space left on device
2023-05-21, 01:41:28.122 [Info   ] [CommonFunc.cpp       CommonFunc::sync                    1192] "Command: sync succeed."
```

Please check if your EFI partition is too small or have enough space to have a new boot entry.

**Comment by babyfengfjx** [comment]
Based on the log, it appears that the EFI partition may be running out of space. You can check the EFI space and, if there are other operating systems installed on the computer, check if multiple kernels have been installed, which may cause the space shortage. You can uninstall some unused kernels to free up space and then try installing again.

**Comment by AhmedKhaledp-0** [comment]
i think isn't   EFI partition cause i deleted my  partition  and created a new one and i had the same problem i think i should get a new disk for deepin

**Reply by BLumia** [reply]
According to the log, it is caused by the EFI partition size. Please ensure that the partition you've deleted is an EFI partition, or check the remaining size of that partition.
